Makefile

```
@@ -202,9 +205,28 @@ local-cluster: # Create a local kind cluster
 release: ## Prepares a tagged release of the operator.
 .PHONY: release
 release:
-	./hack/release/make-release-tag.sh $(OLD_VERSION) $(NEW_VERSION)
+	./hack/release/make-release-tag.sh "$(OLD_VERSION) $(NEW_VERSION)"
```

I think you want this:  
```suggestion
	./hack/release/make-release-tag.sh "$(OLD_VERSION)" "$(NEW_VERSION)"
```